addons/payment/models/payment_transaction.py

```diff
@@ -913,6 +977,12 @@ def _get_sent_message(self):
                 "A transaction with reference %(ref)s has been initiated (%(acq_name)s).",
                 ref=self.reference, acq_name=self.acquirer_id.name
             )
+        elif self.operation == 'refund':
+            formatted_amount = formatLang(self.env, -self.amount, currency_obj=self.currency_id)
```

Comment:
I'd prefer the usage of `tools.format_amount`.
@AntoineVDV do you remember why you used `tools.formatLang` instead of `tools.format_amount` ? :thinking:

Reply:
Nope. Probably inherited from old code.

Reply:
Ok, since it was already used in this file, I made the change to not have another import for format_amount. When putting back format_amount here, do you want me to make the change elsewhere too?

Reply:
Yes, for me you could fully replace formatLang by format_amount :).
The only real change is see between the two methods is the specification of the value as "monetary" by default for format_amount, which makes sense IMHO.